docs/generate_docs.py

```diff
@@ -51,6 +53,11 @@ def patched_import(*args, **kwargs):
         builtins.__import__ = real_import
 
 
+def import_object(name):
+    module, attr = name.rsplit(".", 1)
+    return getattr(importlib.import_module(module), attr)
```

Comment:
@cicdw this is what you'd want to do in the event pr where I mentioned importlib

Reply:
Yup made those changes a while back